infrastructure/main.tf

```diff
@@ -95,7 +51,7 @@ module "db-profile-sync-ref-data-v16" {
   admin_user_object_id = var.jenkins_AAD_objectId
   business_area        = "cft"
   common_tags          = var.common_tags
-  component            = var.component-v16
+  component            = var.component
```

Comment:
if we change this, new db is going to be created?

Reply:
it did not create new db when i merged in demo ,but not taking a chance , reverting it